source/_integrations/geo_location.markdown

```diff
@@ -6,6 +6,7 @@ ha_category:
 logo: geo_location.png
 ha_release: 0.78
 ha_domain: geo_location
+ha_iot_class: Cloud Push
```

Comment:
The `geo_location` integration by itself, is an entity component. It cannot be used by itself, but as a base for other integration to build upon.

Hence, it has no class.

```suggestion
ha_iot_class: ~
```